### PR TITLE
In Link we trust

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
 * Replaced the majority of `a` tags with `Link` wrappers
 * Manually tested and fixed all known broken links
 * Fixed wrong link on `/vuln` view

### CODE REVIEW

**Review by nulls — APPROVED** [review]

**Review by Cheshiriks — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/WelcomeUtils.kt [hunk 1]
```diff
@@ -24,11 +24,13 @@ import web.cssom.*
 /**
  * @param oauthProviders
  * @param primaryColor color of a shield
+ * @param continueLink link for `continue` button
  */
 @Suppress("TOO_LONG_FUNCTION")
 internal fun ChildrenBuilder.inputCredentialsView(
     oauthProviders: List<OauthProviderInfo>,
     primaryColor: Colors,
+    continueLink: String,
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/WelcomeUtils.kt:33]: why do you need an argument if it is always hardcoded?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt [hunk 2]
```diff
@@ -65,8 +62,8 @@ class FallbackView : AbstractView<FallbackViewProps, State>(Style.SAVE_LIGHT) {
                     }
                 }
             } else {
-                a {
-                    href = "${window.location.origin}/"
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt:69]: are you sure? Sounds like it was done intentionally

**Review by akuleshov7 — APPROVED** [review]
looks ok

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/WelcomeUtils.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/WelcomeUtils.kt:33]: Is it always hardcoded?

I guess user unauthorised user could visit both `/vuln/list` and `/projects`

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt (hunk 2 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/FallbackView.kt]: Fixed

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/WelcomeUtils.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/WelcomeUtils.kt:33]: screenshot

**Pull request merged**